components/accreditation/AccreditationProjectCarousel.vue

```diff
@@ -0,0 +1,32 @@
+<script lang="ts" setup>
+// TODO: add functionality
+import { defaultProject } from './default-project'
+import { AccreditationpageDocumentDataProjectsItem } from 'prismicio-types'
+
+interface Props {
+    projects: Pick<AccreditationpageDocumentDataProjectsItem, 'title' | 'location' | 'image' | 'project_link'>[]
+}
+
+withDefaults(defineProps<Props>(), {
+  projects: () => ([defaultProject, { ...defaultProject, title: 'Project Default B' }])
+})
+</script>
+
+<template>
+  <div class="grid lg:flex place-content-center gap-8 pb-32 lg:pb-0">
+    <template v-for="project in projects">
```

Comment:
The case for the `template` here is that the `a` tag is being rendered conditionally with v-if and you can't use v-for and v-if for the same tag. `div` is the parent of the child elements so it doesn't make sense to use v-for there because we don't want to repeat the parent but the child elements. However, this is a kinda WIP for when we do the carousel but I can change it to the below for now as it makes more sense: 

```html
<ul>
  <li v-for="project of projects">
    <a v-if="project.project_link">
    ...
    </a>
  </li>
  ...
</ul>
```